src/main/groovy/de/aaschmid/gradle/plugins/cpd/Cpd.groovy

```diff
@@ -215,7 +214,18 @@ class Cpd extends SourceTask implements VerificationTask, Reporting<CpdReports>
         }
     }
 
-    /**
+    private String asClickableFileUrl(File path) {
+        // File.toURI().toString() leads to an URL like this on Mac: file:/reports/index.html
+        // This URL is not recognized by the Mac console (too few leading slashes). We solve
+        // this be creating an URI with an empty authority.
+        try {
+            return new URI("file", "", path.toURI().getPath(), null, null).toString();
+        } catch (URISyntaxException e) {
+            throw UncheckedException.throwAsUncheckedException(e);
```

Comment:
The import of ```UncheckedException``` is missing as far as I could see and as this is Groovy it will fail at runtime at earliest.